node/index.js

```diff
@@ -105,7 +112,20 @@ function TChannel(options) {
         }
     });
     self.serverSocket.on('error', function onServerSocketError(err) {
-        self.logger.error(self.hostPort + ' server socket error: ' + inspect(err));
+        if (err.code === 'EADDRINUSE') {
```

Comment:
note: we should only wrap EADDRINUSE errors; any other errors will be bubbled up without wrapping context.

Maybe we should wrap them in a generic `TChannelServerError` instead ...